app/helpers/values.js

```diff
@@ -1 +1 @@
-export { default, values } from 'ember-composable-helpers/helpers/values';
+export { default } from 'ember-composable-helpers/helpers/values';
```

Comment:
I took a different approach in #406 here since other helpers are providing re-exports too.